docs/src/main/asciidoc/telemetry-micrometer.adoc

```diff
@@ -0,0 +1,668 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id=telemetry-micrometer]
```

Comment:
@ebullient- If micrometer in Quarkus about more than just metrics, it might be good to be more specific here. We could go deeper too, especially as the content evolves.

```suggestion
[id=telemetry-micrometer-metrics]
```

Reply:
micrometer (this article) is just metrics. There is an eventual larger group for telemetry topics.